client/guidestours/positioning.js

```diff
@@ -75,13 +75,31 @@ const bullseyePositioners = {
 	middle: () => ( {} ),
 };
 
-export function getStepPosition( { placement = 'center', target } ) {
+export function getStepPosition( { placement = 'center', targetSlug } ) {
+	const target = query( '[data-tip-target="' + targetSlug + '"]' )[0];
```

Comment:
In `GuidesTours:quit` we still use `currentTarget` to remove the overlay on quit (step-component-agnostic). Not sure we want to figure out how to get rid of that right now.